src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -839,11 +850,11 @@ public ImmutableArray<T> RemoveRange(ImmutableArray<T> items)
         public ImmutableArray<T> RemoveRange(ImmutableArray<T> items, IEqualityComparer<T> equalityComparer)
         {
             var self = this;
+            self.ThrowNullRefIfNotInitialized();
```

Comment:
:memo: The only case where this changes behavior is when *both* `this` and `items` are default instances of `ImmutableArray<T>` - the change is due to the reversal in the order of checking. Previously this threw an `ArgumentNullException` for this case, but now it throws a `NullReferenceException`.

:thought_balloon: The new behavior appears to be more consistent with other methods in this type.